gensim/matutils.py

```diff
@@ -382,6 +383,110 @@ def cossim(vec1, vec2):
     return result
 
 
+def isbow(vec):
+    """
+    Checks if vector passed is in bag of words representation or not.
+    """
+    if scipy.sparse.issparse(vec):
+        vec = vec.todense().tolist()
+    try:
+        id_, val_ = vec[0]
```

Comment:
In [utils.is_corpus()](https://github.com/piskvorky/gensim/blob/develop/gensim/utils.py#L589), we interpret empty input to mean "yes, this is bag-of-words (return True)".

For consistency, I'd do the same thing here.